src/components/vice/admin/constants.js

```diff
@@ -0,0 +1,57 @@
+export const COMMON_COLUMNS = {
+    NAME: "name",
+    NAMESPACE: "namespace",
+    ANALYSIS_NAME: "analysisName",
+    APP_NAME: "appName",
+    APP_ID: "appID",
+    EXTERNAL_ID: "externalID",
+    USER_ID: "userID",
+    USERNAME: "username",
+    CREATION_TIMESTAMP: "creationTimestamp",
+    EXPAND: "expand",
+};
+
```

Comment:
I kind of disagree here. The organization of the other components seems to be oriented by class/object type (analyses, apps, dashboard, data, etc.), not by page section/layout. Rather than mix the organization between tests and code, I moved the tests so they're in `vice/admin/filter` rather than `admin/vice/filter`.

Reply:
True. But none of the other components have anything related to administration yet. So will there be any non `admin` stuff  in `vice` folder under `src`? If not, I don't think `admin` sub-folder is required. 
On the other hand, My thinking is that by combining all admin related stuff under `admin` folder, it will be easier to navigate code. For example, there will be App administration, tool administration etc.. will be direct sub folders of top level `admin` folder.
It is not a show stopper issue at the moment. Feel free to merge. I will wait for others to pitch in as well.